util/model_output_parser.py

```diff
@@ -60,6 +61,80 @@ def parse_netcdf(file_id, filename, specification):
     ds.close()
     return var_dfs
 
+def parse_grib(file_id, filename, specification):
+    logger.debug("parse GRIB file {}".format(filename))
+    time_dim = specification["time_dim"]
+    horizontal_dims = specification["horizontal_dims"]
+    fill_value_key = specification.get("fill_value_key", None)
+
+    ds_grib = earthkit.data.from_source("file", filename)
+    short_name_excl = specification["var_excl"]
+
+    short_name = np.unique(ds_grib.metadata("shortName"))
+    short_name = short_name[np.where(~np.isin(short_name, short_name_excl))].tolist()
+
+    level_type = np.unique(ds_grib.metadata("typeOfLevel")).tolist()
+    level_type.remove("unknown")
```

Comment:
I think here we should make sure `"unknown"` exists in the `level_type` list to avoid errors:

```suggestion
    if "unknown" in level_type: level_type.remove("unknown")
```

Or you could use a `try` statement, but this seems easier.

Reply:
True, thanks!